resource/decker/support/plugins/handout/handout.js

```diff
@@ -0,0 +1,370 @@
+let Reveal;
+
+let centralSlide;
+
+let handoutSlideMode = false;
+const fakeRevealContainer = document.createElement("div");
+const fakeSlideContainer = document.createElement("div");
+fakeRevealContainer.appendChild(fakeSlideContainer);
+
+function undoAutomaticSlideAdjustments(slideElement) {
+  let slides = slideElement.querySelectorAll("section");
+  for (const slide of slides) {
+    slide.inert = false;
+    slide.hidden = false;
+    slide.removeAttribute("aria-hidden");
+  }
+}
+
+function findAllRealSlides(slidesElement, slideList) {
+  for (const child of slidesElement.childNodes) {
+    if (child.tagName !== "SECTION") continue;
+    if (child.classList.contains("stack")) {
+      for (const subchild of child.childNodes) {
+        if (subchild.tagName !== "SECTION") continue;
+        subchild.previousParent = child;
+        const indices = Reveal.getIndices(subchild);
+        if (indices && indices.h) {
+          child.dataset.hIndex = indices.h;
```

Comment:
why does child get the index of subchild?

Reply:
That was a typo. Fixed.